futures/README.md

```diff
@@ -96,7 +97,11 @@ fn main() {
       // connect() returns a future of an AMQP Client
       // that resolves once the handshake is done
       lapin::client::Client::connect(stream, &ConnectionOptions::default())
-    }).and_then(|client| {
+    }).and_then(|(client, heartbeat_future_fn)| {
```

Comment:
maybe add some comments to explain why the heartbeat needs to run in its own thread